wrapper/pom.xml

```diff
@@ -47,6 +47,122 @@
         </developer>
     </developers>
 
+    <build>
+        <sourceDirectory>src/main/java</sourceDirectory>
+        <testSourceDirectory>src/test/java</testSourceDirectory>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <!--
+                      Bugs in older versions prevent altReleaseDeploymentRepository
+                      and altSnapshotDeploymentRepository from working correctly
+                      https://maven.apache.org/plugins/maven-deploy-plugin/deploy-mojo.html
+                     -->
+                    <artifactId>maven-deploy-plugin</artifactId>
+                    <version>2.8.1</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-release-plugin</artifactId>
+                <version>3.0.0-M4</version>
+                <configuration>
+                    <pushChanges>false</pushChanges>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>exec-maven-plugin</artifactId>
+                <groupId>org.codehaus.mojo</groupId>
+                <configuration>
+                    <executable>./.github/update-version.sh</executable>
```

Comment:
Does this script exist somewhere?

Reply:
Leftovers from previous attempt with maven-release plugin. I shall remove whole config.